app/validators/blocks/block_validator.py

```diff
@@ -7,6 +7,9 @@ class BlockValidator(Validator):
     ANSWER_REFERENCE_INVALID = "Invalid answer reference"
     LIST_REFERENCE_INVALID = "Invalid list reference"
     ANSWER_SELF_REFERENCE = "Invalid answer reference (self-reference)"
+    LIST_NAME_MISSING = "List name defined in action params does not exist"
+    BLOCK_ID_MISSING = "Block id defined in action params does not exist"
```

Comment:
I think `MISSING` implies the property has been omitted (which is covered by the schema), these two errors would be clearer if `MISSING` was replaced with `DOESNT_EXIST`.